.github/workflows/main.yml

```diff
@@ -37,24 +42,16 @@ jobs:
           major_pattern: "(MAJOR)"
           minor_pattern: "(MINOR)"
 
-      - name: Package
-        run: dotnet pack --configuration Release  -p:Version=${{ steps.sdk-version.outputs.version }}
-
-      # Publish to github
-      - name: Add nuget sources
-        run: dotnet nuget add source --username USERNAME --password ${{ secrets.READ_PACKAGES_TOKEN }} --store-password-in-clear-text --name github "https://nuget.pkg.github.com/threesigmaxyz/index.json"
-
-      - name: Publish Commons SDK
-        run: dotnet nuget push "src/StarkEx.Commons.SDK/bin/Release/*.nupkg" --api-key ${{ secrets.GITHUB_TOKEN }} --source "github"
-      - name: Publish Crypto SDK
-        run: dotnet nuget push "src/StarkEx.Crypto.SDK/bin/Release/*.nupkg" --api-key ${{ secrets.GITHUB_TOKEN }} --source "github"
-      - name: Publish Client SDK
-        run: dotnet nuget push "src/StarkEx.Client.SDK/bin/Release/*.nupkg" --api-key ${{ secrets.GITHUB_TOKEN }} --source "github"
-
-      # Publish to nuget.org
-      - name: Publish Commons SDK
-        run: dotnet nuget push "src/StarkEx.Commons.SDK/bin/Release/*.nupkg" --api-key ${{ secrets.NUGET_API_KEY }} --source "nuget.org"
-      - name: Publish Crypto SDK
-        run: dotnet nuget push "src/StarkEx.Crypto.SDK/bin/Release/*.nupkg" --api-key ${{ secrets.NUGET_API_KEY }} --source "nuget.org"
-      - name: Publish Client SDK
-        run: dotnet nuget push "src/StarkEx.Client.SDK/bin/Release/*.nupkg" --api-key ${{ secrets.NUGET_API_KEY }} --source "nuget.org"
+      - name: Setup Node.js
```

Comment:
Do we still need the `Tag version` step?

Reply:
I don't understand the question. What `Tag version` step are you referencing?

Reply:
I did remove it but forgot to push that last change. It is now removed